oxide-vpc/tests/common/mod.rs

```diff
@@ -120,7 +120,7 @@ pub fn g1_cfg() -> VpcCfg {
             gateway_ip: "fd00::1".parse().unwrap(),
             snat: Some(SNat6Cfg {
                 external_ip: "2001:db8::1".parse().unwrap(),
-                ports: 1025..=4096,
+                ports: 4097..=8192,
```

Comment:
Just curious, why this change?

Reply:
While adding the `guest_to_internet_ipv6` test I had initially left one assert referring to the v4 SNAT config. To hopefully reduce those kinds of mix ups I made this change to use different ranges for IPv4 vs IPv6.

Also, previously `g1_cfg` and `g2_cfg` were using the same port range and IP for the v6 SNAT config.